### DISCUSSION TITLE
[Documentation] Shell command for getting started

### DISCUSSION THREAD

**Discussion by w3dg** [vercel/next.js#10939] [Help] [answered]
In the getting started docs the Setup section says :-
```sh 
npm init next-app 
yarn create next-app 
```
instead of **`create-next-app`**  
while a line above it says create-react-app . 
I think it is wrong?

**Comment by rafaelalmeidatk** [comment, marked as answer]
Both commands will use `create-next-app`:
- https://docs.npmjs.com/cli/init
- https://classic.yarnpkg.com/en/docs/cli/create/

**Comment by w3dg** [comment]
Ok thanks ! Besides can one use `npx create-next-app` ?

**Reply by rafaelalmeidatk** [reply]
Yep, in the end they all do the same thing
